app/templates/Piwik.twig

```diff
@@ -0,0 +1,26 @@
+<!-- Piwik -->
+<script type="text/javascript">
+	var _paq = _paq || [];
+	_paq.push( [ 'trackPageView' ] );
+	_paq.push( [ 'enableLinkTracking' ] );
+
+	{% for trackingEvent in piwikTracking %}
+	_paq.push( {$ trackingEvent | json_encode() | raw $} );
+	{% endfor %}
+
+	( function () {
+		var u = "{$ piwik.baseUrl $}";
+		_paq.push( [ 'setTrackerUrl', u + 'piwik.php' ] );
+		_paq.push( [ 'setSiteId', {$ piwik.siteId $} ] );
+		var d = document,
+			g = d.createElement( 'script' ),
+			s = d.getElementsByTagName( 'script' )[ 0 ];
+		g.type = 'text/javascript';
+		g.async = true;
+		g.defer = true;
+		g.src = u + 'piwik.js';
+		s.parentNode.insertBefore( g, s );
+	} ) ();
+</script>
+<noscript><p><img src="{$ piwik.baseUrl $}piwik.php?idsite={$ piwik.siteId $}" style="border:0;" alt="" /></p></noscript>
```

Comment:
Not sure, if it creates a link break, but it's definitely not needed. I copied it from the piwik installation's gimme-the-snippet-thingy.